axelor-base/src/main/java/com/axelor/apps/base/service/administration/IndicatorGeneratorGroupingService.java

```diff
@@ -101,8 +105,9 @@ public void export(IndicatorGeneratorGrouping indicatorGeneratorGrouping) throws
     }
 
     try {
+      String dataExportDir = appService.getDataExportDir();
```

Comment:
Move this line of code to starting of the method

Suppose, user had configured the dataExportDir but not configured the indicatorGeneratorGrouping.getPath() so in that case we also don't have to export
For that update CsvTool.csvWriter() parameter:
pass filePath null if indicatorGeneratorGrouping.getPath() is null so it will throw IOException and we already have catch block for it